### PR TITLE
Added mistral provider documentation

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary
- Added initial documentation of mistral ai documentation

### CODE REVIEW

File: content/ai-gateway/providers/mistral.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+pcx_content_type: get-started
+---
+
+{{<heading-pill style="beta">}}Mistral AI Platform{{</heading-pill>}}
+
+[Mistral AI Platform](https://mistral.ai) helps you build quickly with Mistral's advanced AI models.
+
+## Endpoint
+
+`https://gateway.ai.cloudflare.com/v1/{account_tag}/{gateway}/mistral`
```
Comment (kodster28) [inline content/ai-gateway/providers/mistral.md]:
```suggestion
`https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/mistral`
```

Comment (kodster28) [reply content/ai-gateway/providers/mistral.md]: We generally prefer `Account ID` as opposed to `Account tag`. Technically, `tag` is more accurate from an internal perspective, but `id` corresponds to what a customer sees in their account.

Realize that's a touch pedantic, but think it helps customers know what we're talking about.

File: content/ai-gateway/providers/mistral.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## What you need
+
+When making requests to the Mistral AI Platform, you will need:
+
+- Mistral AI account ID
```
Comment (kodster28) [inline content/ai-gateway/providers/mistral.md:20]:
```suggestion
- AI Gateway Account ID
- AI Gateway gateway name
- Mistral AI account ID
```

File: content/ai-gateway/providers/mistral.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- Mistral AI gateway name
+- Mistral AI Platform API token
+- Mistral AI model name
+
+## URL structure
+
+Your new base URL will use the data above in this structure: `https://gateway.ai.cloudflare.com/v1/{account_tag}/{gateway}/mistral/`.
```
Comment (kodster28) [inline content/ai-gateway/providers/mistral.md]: Fix the `account_tag` / `gateway` to match above.

**Review by kodster28 — APPROVED** [review]
Should wait for @kathayl's feedback as well (she's great at finding stuff even I miss).

File: content/ai-gateway/providers/mistral.md (hunk 2 above)
Comment (kathayl) [reply content/ai-gateway/providers/mistral.md:20]: 4 things should be

- AI Gateway account ID
- AI Gateway gateway name
- Mistral API key
- Mistral model name

**Comment by kathayl** [comment]
Curious why we chose to refer to Mistral AI as Mistral AI Platform. Did you find that that was the way they typically refer to themselves?

File: content/ai-gateway/providers/mistral.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+Then you can append the endpoint you want to hit, for example: `v1/chat/completions`
+
+So your final URL will come together as: `https://gateway.ai.cloudflare.com/v1/{account_tag}/{gateway}/mistral/v1/chat/completions`.
```
Comment (kathayl) [inline content/ai-gateway/providers/mistral.md]: Fix the `account_tag` /` gateway` to match above.

File: content/ai-gateway/providers/mistral.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Examples
+
+### cURL
+
+```bash
+---
+header: Example fetch request
+---
+curl -X POST https://gateway.ai.cloudflare.com/v1/ACCOUNT_TAG/GATEWAY/mistral/v1/chat/completions \
```
Comment (kathayl) [inline content/ai-gateway/providers/mistral.md]: Fix the `account_tag` / `gateway` to match above.

**Comment by daisyfaithauma** [comment]
> Curious why we chose to refer to Mistral AI as Mistral AI Platform. Did you find that that was the way they typically refer to themselves?



> Curious why we chose to refer to Mistral AI as Mistral AI Platform. Did you find that that was the way they typically refer to themselves?



> Curious why we chose to refer to Mistral AI as Mistral AI Platform. Did you find that that was the way they typically refer to themselves?

No reason actually. I think it sounds better without Platform.

**Pull request merged**